### ISSUE TITLE
Effect size standard error bars span zero 

### ISSUE THREAD

**Issue by eam12** [FrederickHuangLin/ANCOMBC#87]
I suppose this is more of a stats question than anything else, but I have been using ANCOM-BC and discovered that the vast majority of my differentially abundant taxa (q-val < 0.05) have a standard error that spans zero (see image for example). Is this something I should be concerned about or does it just indicate there is a lot of variation in the dataset? Thank you so much!
![example](https://user-images.githubusercontent.com/36485497/187292986-6beac570-033b-4044-9fd1-2fb649019c33.png)

**Comment by FrederickHuangLin** [comment]
Hi @eam12,

Can you check the `zero_ind` table and see if these differentially abundant taxa (which have standard errors spaning around zero) are considered as structural zeros?

Best,
Huang

**Comment by eam12** [comment]
Yes, it does appear that the differentially abundant taxa with SE spanning zero are structural zeros. So would the most appropriate thing be to simply remove any differentially abundant taxa identified as structural zeros from further analysis?

**Comment by FrederickHuangLin** [comment]
Hi @eam12,

I am pleased to share with you that the major update for `ANCOMBC` package has been completed. Here are some highlighted new features:

- Add `ancombc2` function, which supports pairwise test and trend test for both cross-sectional and repeated measurements data
- Add sensitivity analysis for pseudo-count addition
- Fully support the `(Tree)SummarizedExperiment` class
- A more user-friendly output layout

We also added the following clarifications for structural zeros: 

> A taxon is considered to have structural zeros in some (>=1) groups if it is completely (or nearly completely) missing in these groups. For instance, suppose there are three groups: g1, g2, and g3. If the counts of taxon A in g1 are 0, but they are nonzero in g2 and g3, then taxon A will be considered to contain structural zeros in g1. In this example, taxon A is declared to be differentially abundant between g1 and g2, g1 and g3, and consequently, it is globally differentially abundant with respect to this group variable. Such taxa are not further analyzed using ANCOM-BC2, but the results are summarized in the overall summary.

Basically, we recommend separate discussions on taxa with structural zeros (**NOT** detected by ANCOM-BC or ANCOM-BC2) and taxa without structural zeros (detected by ANCOM-BC or ANCOM-BC2).

I just pushed the changes to the Bioconductor branches. It might take a few business days for the package to become available (but the `devel` version is ready here!).

Best,
Huang

**Comment by eam12** [comment]
Wonderful! Thank you so much!
